.github/workflows/ci.yml

```
@@ -166,6 +180,7 @@ jobs:
 
       - name: "Cargo test"
         run: |
+          export UV_INDEX_URL="http://localhost:80/simple"
```

```suggestion
          export UV_INDEX_URL="http://127.0.0.1:80/simple"
```
https://www.youtube.com/watch?v=98SYTvNw1kw

and with @samypr100 suggestion
```suggestion
          export UV_INDEX_URL="http://127.0.0.1/simple"
```